src/workerd/api/sockets.c++

```diff
@@ -58,8 +59,9 @@ jsg::Ref<Socket> setupSocket(
   closeFulfiller.promise.markAsHandled();
 
   auto result = jsg::alloc<Socket>(
-      js, kj::mv(readable), kj::mv(writable), kj::mv(closeFulfiller), kj::mv(connDisconnPromise),
-      kj::mv(options), kj::mv(tlsStarter), isSecureSocket, kj::mv(domain));
+      js, kj::mv(refcountedConnection), kj::mv(readable), kj::mv(writable), kj::mv(closeFulfiller),
+      kj::mv(connDisconnPromise), kj::mv(options), kj::mv(tlsStarter), isSecureSocket,
+      kj::mv(domain));
```

Comment:
Just a nit... I really dislike putting arguments all together like this as opposed to putting each on a separate line specifically because adding/removing shows a bigger delta.

```
auto result = jsg::alloc<Socket>(js,
                                 kj::mv(readable),
                                 kj::mv(writable),
                                 kj::mv(closeFulfiller),
                                 //....);
```

When an argument is added/removed/edited the delta shows just that single change.

Anyway, obviously not blocking here.

Reply:
Fair argument. But I will compromise: i'll put each argument on one line but won't align with the `(` (doing this leads to bigger change deltas and subjectively I dislike having to indent things so much).